.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "secp256r1/besu-native-ec"]
 	path = secp256r1/besu-native-ec
 	url = git@github.com:ConsenSys/besu-native-ec.git
+[submodule "ipa-multipoint/ipa_multipoint"]
```

Comment:
you no longer need `ipa_multipoint` as a submodule, as it's referenced by `ipa_multipoint_jni` in its `Cargo.toml`.